redhat-developer/odo
# Add service catalog test that tests a spring boot application with a postgresql db

## What is the purpose of this change? What does it change?
Add test that tests a spring boot application with a postgresql db. 
This has been requested by various people that would like to be able to test a spring boot application that uses a database, without actually having to know the details of spring boot development

## Was the change discussed in an issue?
No, but it has been mentioned in team meetings

## How to test changes?
`RUN_ALL_SERVICE_TESTS=true make test-service-e2e` should run the new tests along with the existing service catalog tests

`make test-service-e2` (which is what Travis uses) will should not execute the new tests
